.github/workflows/samples.yml

```diff
@@ -15,17 +15,18 @@ jobs:
         example:
           - examples/getting-started-typescript
           - examples/custom-receiver
+          - examples/oauth-express-receiver
     steps:
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node-version }}
       - uses: actions/checkout@v4
-      - name: Install dependencies and link bolt-js
-        run: npm i && npm link .
-      - name: Install example dependencies and link bolt-js
+      - name: Package the latest changes to bolt-js
+        run: npm i && npm pack . && rm -rf node_modules
+      - name: Install example dependencies for testing
         working-directory: ${{ matrix.example }}
-        run: npm i && npm link @slack/bolt
+        run: npm i && npm i ../../slack-bolt-*.tgz
```

Comment:
Some TILs are included here, though I believe this can still be improved:

- `npm pack` will package `@slack/bolt` as if it's going to be released
- `node_modules` of the bolt-js repo **must** be removed or nested directories can make use of these pacakges

The `*.tgz` file seems ideal for testing in a production environment like this since it lets us build bolt-js then remove its `node_modules` before testing a sample!

I'm not thrilled with the relative paths here, but am interested in if this is an approach we're alright taking?